owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -596,13 +599,13 @@ abstract class DrawerActivity : ToolbarActivity() {
         // - ACCOUNT_LIST_CHANGED = true
         // - RESULT_OK
         if (requestCode == ACTION_MANAGE_ACCOUNTS && resultCode == Activity.RESULT_OK && data!!.getBooleanExtra(
-                ManageAccountsActivity.KEY_ACCOUNT_LIST_CHANGED,
+                KEY_ACCOUNT_LIST_CHANGED,
                 false
             )
         ) {
 
             // current account has changed
-            if (data.getBooleanExtra(ManageAccountsActivity.KEY_CURRENT_ACCOUNT_CHANGED, false)) {
+            if (data.getBooleanExtra(KEY_CURRENT_ACCOUNT_CHANGED, false)) {
```

Comment:
You will have to add here the name of the class for the app to compile since you will put the constants in a `companion object`, don't forget to check the app continues working correctly after performing changes!

Reply:
Didn't have to, I created a companion object so this wasn't necessary 😎

Reply:
This wasn't necessary because Android Studio imported it for you automatically, but if instead of importing it at the beginning of the class you want to import it in-body, you have to add the name of the class 👍